backprojection.cpp

```diff
@@ -248,12 +248,28 @@ class BackprojectionGenerator : public Halide::Generator<BackprojectionGenerator
             Var x_vo{"x_vo"}, x_vi{"x_vi"};
             Var y_vo{"y_vo"}, y_vi{"y_vi"};
             Var pixeli{"pixeli"}, block{"block"};
-            win_sample.compute_root()
+            win_sample.taylor.compute_root()
                       .vectorize(sample, vectorsize)
                       .parallel(sample, blocksize);
-            win_pulse.compute_root()
+            win_sample.w.compute_root()
+                      .split(sample, sample_vo, sample_vi, 4)
+                      .vectorize(sample_vi)
+                      .parallel(sample_vo);
```

Comment:
pedantic: fix indenting here and elsewhere to align with the `compute_root` call.